src/Annotation/BuilderAbstractFactory.php

```diff
@@ -73,7 +71,7 @@ public function canCreate(ContainerInterface $container, $requestedName)
      *
      * @return array
```

Comment:
PHP Code Sniffer keeps `@return array` notation to let be more specific if the developer is willing to be so